ui/src/childinfo/ChildInfo.js

```diff
@@ -6,10 +6,9 @@ import './ChildInfo.css';
 export default class ChildInfo extends Component {
   constructor(props){
       super();
```

Comment:
don't we need to pass in props in super?

Reply:
Not totally sure but Ill add that in